test/quantum/job_broadcaster_test.exs

```diff
@@ -84,6 +84,10 @@ defmodule Quantum.JobBroadcasterTest do
         TestScheduler.delete_job(broadcaster, active_job.name)
 
         assert_receive {:received, {:remove, ^active_job_name}}
+
+        refute Enum.any?(TestScheduler.jobs(broadcaster), fn {key, _} ->
```

Comment:
what about:
```elixir
refute %{^active_job_name => _} = TestScheduler.jobs(broadcaster)
```
Same below.

Reply:
Because of BC reasons, this is sadly not a map, but a list containing a tuple of name & job.